### PR TITLE
test(db pull): remove duplicated tests

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/schema-team/issues/407

Note that `db pull` tests feels much faster locally, since the split, local run:
```
Test Suites: 10 passed, 10 total
Tests:       1 skipped, 82 passed, 83 total
Snapshots:   470 passed, 470 total
Time:        26.46 s
```

Since https://github.com/prisma/prisma/issues/17582 we do not need to test the rendering of the introspection warnings in the CLI, since they are already tested in prisma-engines. We only need an integration test.

Removed tests were added in prisma-engines from the following PRs: https://github.com/prisma/prisma-engines/pull/3622 https://github.com/prisma/prisma-engines/pull/3657 https://github.com/prisma/prisma-engines/pull/3849 https://github.com/prisma/prisma-engines/pull/3859 https://github.com/prisma/prisma-engines/pull/3871 https://github.com/prisma/prisma-engines/pull/3877 https://github.com/prisma/prisma-engines/pull/3873

### CODE REVIEW

File: packages/client/src/__tests__/integration/errors/multi-schema/test.ts [hunk 1]
```diff
@@ -14,9 +14,9 @@ const title = faker.name.jobTitle()
 const newEmail = faker.internet.email()
 const newTitle = faker.name.jobTitle()
 
-describe('multi-schema', () => {
```
Comment (Jolg42) [inline packages/client/src/__tests__/integration/errors/multi-schema/test.ts:17]: Small rename along the way to make everything consistent

File: packages/migrate/src/__tests__/DbPull/postgresql-views.test.ts [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+      expect(tree).toMatchInlineSnapshot(`
+        [
+          views/extraneous-file.sql,
+        ]
+      `)
+    })
+  })
+})
+
+// TODO remove this once ported to prisma-engines
```
Comment (Jolg42) [inline packages/migrate/src/__tests__/DbPull/postgresql-views.test.ts:528]: Actually it's already tested in `re_intro_keeps_the_field_map` https://github.com/prisma/prisma-engines/blob/a9691be293caff3232d970c1150a952b03f7a0fd/migration-engine/introspection-engine-tests/tests/views/postgresql.rs

Comment (Jolg42) [reply packages/migrate/src/__tests__/DbPull/postgresql-views.test.ts]: So I removed it now

**Review by jkomyno — APPROVED** [review]
I would have preferred a smaller diff (though I understand this is a "CMD+F and replace" case), but this is LGTM

**Review by janpio — COMMENTED** [review]
👍

File: packages/client/src/__tests__/integration/errors/multi-schema/test.ts (hunk 1 above)
Comment (Druue) [reply packages/client/src/__tests__/integration/errors/multi-schema/test.ts:17]: Yeah, probably update it to `multischema/test.ts`

**Review by Druue — APPROVED** [review]
✨

File: packages/client/src/__tests__/integration/errors/multi-schema/test.ts (hunk 1 above)
Comment (Jolg42) [reply packages/client/src/__tests__/integration/errors/multi-schema/test.ts:17]: indeed, I'll do this after

**Pull request merged**